### ISSUE TITLE
2.1.1: uninitialized constant ActivityNotification::SubscriptionsApiWithDeviseController::DeviseTokenAuth

### ISSUE THREAD

**Issue by jules27** [simukappu/activity_notification#129]
Hi! I upgraded this gem from 2.0.0 to 2.1.1 because I wanted to override mail headers like `reply-to` and `from`. I had no problem with the version upgrade in my local environment but ran into an issue when I deploy to a production environment.

### Steps to reproduce

Release to a production environment with `activity_notification` version 2.1.1.

### Expected behavior

The release should succeed.

### Actual behavior

The release fails with the traceback (clipped):

```
  NameError: uninitialized constant ActivityNotification::SubscriptionsApiWithDeviseController::DeviseTokenAuth
    /home/deploy/server/shared/bundle/ruby/2.6.0/gems/activity_notification-2.1.1/app/controllers/activity_notification/subscriptions_api_with_devise_controller.rb:4:in `'
    /home/deploy/server/shared/bundle/ruby/2.6.0/gems/activity_notification-2.1.1/app/controllers/activity_notification/subscriptions_api_with_devise_controller.rb:3:in `'
    /home/deploy/server/shared/bundle/ruby/2.6.0/gems/activity_notification-2.1.1/app/controllers/activity_notification/subscriptions_api_with_devise_controller.rb:1:in `'
    /home/deploy/server/shared/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require'
    /home/deploy/server/shared/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `block in require_with_bootsnap_lfi'
    /home/deploy/server/shared/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
    /home/deploy/server/shared/bundle/ruby/2.6.0/gems/bootsnap-1.4.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:21:in `require_with_bootsnap_lfi'
```

### System configuration

**activity_notification gem version**: 2.1.1
**Rails version**: 6.0.0
**Ruby Version**: 2.6.3 and 2.6.5
**ORM (ActiveRecord, Mongoid or Dynamoid)**: ActiveRecord

I'm unable to reproduce this error in my local environment but am happy to try any fixes/suggestions on my production environment.

**Comment by simukappu** [comment]
Hi @jules27,
Are you using *SubscriptionsApiWithDeviseController*? Can you share your *config/routes.rb*?

**Comment by Eth3rnit3** [comment]
I also had this problem when I try to launch my application with `config.eager_load = true` when I don't use api controllers. 
I submit a patch to you

**Comment by simukappu** [comment]
@Eth3rnit3, thank you for your feedback! I could reproduce this issue and added the patch.

**Comment by simukappu** [comment]
Just released as v2.1.2. Thank you for your contribution!

**Comment by jules27** [comment]
Amazing, thank you both!!
